tests/rados/test_reads_balancer.py

```diff
@@ -278,8 +282,175 @@ def run(ceph_cluster, **kw):
                         log.error(f"An error occurred but was expected: {e}")
                         log.debug("Could not change the primary for EC pools. Pass")
                     break
-                log.info("Verified the Online commands to change acting set")
-                return 0
+
+            log.info("Verified the Online commands to change acting set")
+            return 0
+
+        if config.get("offline_command_verification"):
+            log.info(
+                "Verifying the usage of offline commands to change the acting set of the pool"
+            )
+            for pool_name in existing_pools:
+                pool_details = rados_obj.get_pool_details(pool=pool_name)
+                log.debug(f"Selected pool name : {pool_name}")
+
+                # Testing on replicated pools
+                if not pool_details["erasure_code_profile"]:
+                    pool_details = rados_obj.get_pool_details(pool=pool_name)
```

Comment:
This line seems to be redundant, pool_details has already been fetched in line# 294

Reply:
yes. removed!